README.md

```diff
@@ -121,22 +113,45 @@ You can then add
  ```
 to your Swift code to access the functionality in the `HomomorphicEncryption` library.
 
-See the example [Snippets](https://github.com/apple/swift-homomorphic-encryption/tree/main/Snippets) for examples of using `HomomorphicEncryption`.
-To run the `EncryptionParametersSnippet` example, run
+> [!NOTE]
+> If you are using Swift Homomorphic Encryption for research, please cite using the
+> [CITATION.cff](CITATION.cff) file.
+
+#### Examples
+See the [Snippets](https://github.com/apple/swift-homomorphic-encryption/tree/main/Snippets) for examples using `HomomorphicEncryption`.
+To run the `EncryptionParametersSnippet`, run
 ```
 swift run -c release EncryptionParametersSnippet
 ```
 
+### Supported Platforms
+Swift Homomorphic Encryption aims to support all of the platforms where Swift is supported. Currently, it is developed and tested on macOS and Linux, and is known to support 64-bit processors with little-endian memory representation and the following operating system versions:
+* Ubuntu 22.04
+* macOS 15.0 with Xcode 16.1
```

Comment:
Not the same as in [UsingSwiftHomomorphicEncryption.md](https://github.com/apple/swift-homomorphic-encryption/pull/124/files#diff-90ab7ea396b3844ae01e97333b2df69a0ad988925f5888b6b754570a3eace683)